lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -3,6 +3,190 @@
+     * - authority: the authority URL for your application.
+     * - redirect_uri: the uri of your application registered in the portal.
+     *
+     * In Azure AD, authority is a URL indicating the Azure active directory that MSAL uses to obtain tokens.
+     * It is of the form https://login.microsoftonline.com/&lt;Enter_the_Tenant_Info_Here&gt;.
+     * If your application supports Accounts in one organizational directory, replace "Enter_the_Tenant_Info_Here" value with the Tenant Id or Tenant name (for example, contoso.microsoft.com).
+     * If your application supports Accounts in any organizational directory, replace "Enter_the_Tenant_Info_Here" value with organizations.
+     * If your application supports Accounts in any organizational directory and personal Microsoft accounts, replace "Enter_the_Tenant_Info_Here" value with common.
+     * To restrict support to Personal Microsoft accounts only, replace "Enter_the_Tenant_Info_Here" value with consumers.
+     *
+     * In Azure B2C, authority is of the form https://&lt;instance&gt;/tfp/&lt;tenant&gt;/&lt;policyName&gt;/
+     *
+     * @param {@link (Configuration:type)} configuration object for the MSAL PublicClientApplication instance
+     */
+    constructor(configuration: Configuration) {
+        // Set the configuration
+        this.config = buildConfiguration(configuration);
+
+        this.authModule = new msalAuth.AuthorizationCodeModule({
+            auth: this.config.auth,
+            cryptoInterface: null,
+            networkInterface: null,
+            storageInterface: null
+        });
+    }
+
+    // #region Redirect Flow
+
+    /**
+     * Set the callback functions for the redirect flow to send back the success or error object.
+     * @param {@link (authCallback:type)} authCallback - Callback which contains
+     * an AuthError object, containing error data from either the server
+     * or the library, depending on the origin of the error, or the AuthResponse object 
+     * containing data from the server (returned with a null or non-blocking error).
+     */
+    handleRedirectCallback(authCallback: authCallback): void {
+        throw new Error("Method not implemented.");
+    }
+
+    /**
+     * Use when initiating the login process by redirecting the user's browser to the authorization endpoint.
+     * @param {@link (AuthenticationParameters:type)}
+     */
+    loginRedirect(request: msalAuth.AuthenticationParameters): void {
+        throw new Error("Method not implemented.");
+    }
+
+    /**
+     * Use when you want to obtain an access_token for your API by redirecting the user's browser window to the authorization endpoint.
+     * @param {@link (AuthenticationParameters:type)}
+     *
+     * To renew idToken, please pass clientId as the only scope in the Authentication Parameters
```

Comment:
this statement here feels like we may want a separate function to explicitly to renew an idToken, rather than using parameters as a functionality-select.

Reply:
This comment should be in ATS - fix @pkanher617